packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## NEXT
+## 0.3.6
```

Comment:
Please see the Dart version guidance for package versions <1.0 linked from the checklist; this is a minor version in Dart conventions, and the PR is just a bugfix.